packages/@aws-cdk/aws-eks-v2-alpha/lib/cluster.ts

```diff
@@ -1109,6 +1222,25 @@ export class Cluster extends ClusterBase {
         authenticationMode: 'API',
         bootstrapClusterCreatorAdminPermissions: props.bootstrapClusterCreatorAdminPermissions,
       },
+      computeConfig: {
+        enabled: !autoModeEnabled ? false : true,
+        // If the computeConfig enabled flag is set to false when creating a cluster with Auto Mode,
+        // the request must not include values for the nodeRoleArn or nodePools fields.
+        nodePools: !autoModeEnabled ? undefined : props.compute?.nodePools ?? ['system', 'general-purpose'],
+        nodeRoleArn: !autoModeEnabled ? undefined : props.compute?.nodeRole?.roleArn ?? this.addNodePoolRole(`${id}nodePoolRole`).roleArn,
+      },
+      storageConfig: {
+        blockStorage: {
+          enabled: !autoModeEnabled ? false : true,
```

Comment:
can `blockStorage` be `disabled` when auto mode is `enabled`?

Reply:
No, all the three booleans have to be all enabled or all disabled. We learn this from cloudformation errors.